### PR TITLE
Major Upgrade docs prep: exchange-operators section

### PR DESCRIPTION
**Pull request by qwadratic**
- Create index for Exchange Operators
- Fix broken link to Archive Node instructions

### CODE REVIEW

File: docs/exchange-operators/index.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - historical data
+  - blockchain integration
+  - explorer
+  - wallet
+  - archive node
+  - graphQL
+  - rosetta
+---
+
+# Introduction
```
Comment (barriebyron) [inline docs/exchange-operators/index.mdx:15]:
```suggestion
# Exchange Operators
```

File: docs/exchange-operators/index.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Exchange operators are network participants who run mina nodes on a Mina network, but usually don't produce blocks.
+
+Instead of running Mina Node and Mina Archive, we recommend to run Rosetta API, designed specifically for exchange operators needs.
+
+It implements an [open-source blockchain-agnostic specification](https://docs.cloud.coinbase.com/rosetta/docs/welcome) designed by Coinbase.
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/exchange-operators/index.mdx:21]: the docs search works best with only 2 levels of "sections" 
Let's move the Rosetta API docs in one level

File: sidebars.js [hunk 3]
```diff
@@ -512,6 +516,7 @@ module.exports = {
             type: 'doc',
             id: 'exchange-operators/rosetta/index',
```
Comment (barriebyron) [inline sidebars.js:517]: see my earlier comments about the section levels... I strongly recommend moving the Rosetta API in one level so it's not unnecessarily nested

**Review by barriebyron — COMMENTED** [review]
see comments about bringing the Rosetta API docs in one level

**Comment by qwadratic** [comment]
@barriebyron regarding having the links inside the index.md - I just followed the approach that @halsaphi took in Node Operators docs update. The category index has links to sub-articles.

File: docs/exchange-operators/rosetta/build-from-sources.mdx [hunk 4]
```diff
@@ -21,7 +21,7 @@ If you already have mina-archive up and running, you can also build and run the
 
 :::tip
 
-To run mina daemon and mina-archive natively, follow the [Archive Node page](node-operators/archive-nodes/getting-started) instructions.
+To run mina daemon and mina-archive natively, follow the [Archive Node](/node-operators/archive-nodes/getting-started) instructions.
```
Comment (barriebyron) [inline docs/exchange-operators/rosetta/build-from-sources.mdx:24]: depending on 
```suggestion
To run mina daemon and mina-archive natively, follow the [Archive Node](/node-operators/archive-node/getting-started) instructions.
```

**Comment by qwadratic** [comment]
@barriebyron can you please take one more look? I think its GTG now

**Review by barriebyron — APPROVED** [review]
although the docs structure is nested more than the recommended depth of 2, this PR is approved. Let's wait for #815 before merging, so that all links to the /archive-node/ docs will work 

Thanks @qwadratic

**Pull request merged**